f2/src/forms/ContactInfoForm.js

```diff
@@ -34,7 +37,7 @@ export const validate = (values) => {
     errors.email = 'contactinfoForm.email.warning'
   }
 
-  if (!values.phone) {
+  if (!new RegExp(phoneRegex).test(values.phone)) {
```

Comment:
This validation is only applied to the full report. If I submit FYI report I can still enter "a" as the phone number.

Reply:
Yeah, you are right. I didn't check FYI flow, I'll check that.